config/initializers/string.rb

```diff
@@ -9,3 +9,9 @@ def secure_compare(other)
     res == 0
   end
 end
+
+class NilClass
+  def secure_compare(_)
```

Comment:
o_O

I believe there must be a better solution than monkey-patching `NilClass`.